rust/log/src/grpc_log.rs

```diff
@@ -639,19 +659,43 @@ impl GrpcLog {
         }
     }
 
-    pub(super) async fn seal_log(
+    pub async fn seal_log(
         &mut self,
-        tenant: &str,
         collection_id: CollectionUuid,
     ) -> Result<(), GrpcSealLogError> {
+        // NOTE(rescrv):  Seal log only goes to the go log service, so use the classic connection.
         let _response = self
-            .client_for(tenant, collection_id)?
+            .client
             .seal_log(chroma_proto::SealLogRequest {
                 collection_id: collection_id.to_string(),
             })
             .await?;
         Ok(())
     }
+
+    pub async fn migrate_log(
+        &mut self,
+        collection_id: CollectionUuid,
+    ) -> Result<(), GrpcMigrateLogError> {
+        // NOTE(rescrv): Migrate log only goes to the rust log service, so use alt client assigner.
+        if let Some(client_assigner) = self.alt_client_assigner.as_mut() {
+            let mut client = client_assigner
+                .clients(&collection_id.to_string())?
+                .drain(..)
+                .next()
+                .ok_or(ClientAssignmentError::NoClientFound(
+                    "Impossible state: no client found for collection".to_string(),
+                ))?;
+            client
+                .migrate_log(chroma_proto::MigrateLogRequest {
+                    collection_id: collection_id.to_string(),
+                })
+                .await?;
+            Ok(())
+        } else {
+            Err(GrpcMigrateLogError::NotSupported)
```

Comment:
this maps to `ErrorCodes::Unimplemented` but is that indicative of the underlying error? Should we have a separate error to communicate `NoClientFound`?